src/core/lib/slice/b64.cc

```diff
@@ -65,7 +65,7 @@ char* grpc_base64_encode(const void* vdata, size_t data_size, int url_safe,
   return result;
 }
 
-size_t grpc_base64_estimate_encoded_size(size_t data_size, int url_safe,
+size_t grpc_base64_estimate_encoded_size(size_t data_size, int /*url_safe*/,
```

Comment:
This isn't implementing an interface, so if the parameter is not used, it should probably just be removed.